native-navigation/android/src/main/java/com/mediamonks/rnnativenavigation/factory/ModalFragment.java

```diff
@@ -45,6 +78,18 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         View.generateViewId();
         view.setId(View.generateViewId());
 
+        ViewTreeObserver viewTreeObserver = view.getViewTreeObserver();
+        viewTreeObserver.addOnGlobalFocusChangeListener(new ViewTreeObserver.OnGlobalFocusChangeListener() {
+            @Override
+            public void onGlobalFocusChanged(View oldFocus, View newFocus) {
+                if (newFocus != null) {
+                    // DialogsFragment views don't bubble alphanumeric key events up to the containing Activity, because their containing PhoneVWindow discards them.
+                    // Here we forward them forcefully in order to allow development shortcuts on the emulator
+                    newFocus.setOnKeyListener(_forwardKeyListener);
```

Comment:
Shouldn't oldFocus remove the keylistener?

Reply:
It can, but it doesn't need to, because `setOnKeyListener()` is just a simple setter (ie. it doesn't notify observers or or add the listener to a listener list).
I can add it though.

Reply:
If you say so it's fine.. maybe a comment to prevent future confusion.